src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/TestFiles/DefaultRazorSemanticTokenInfoServiceTest/GetSemanticTokens_Razor_NestedTextDirectives.semantic.txt

```diff
@@ -0,0 +1,20 @@
+//line,characterPos,length,tokenType,modifier
+0 0 1 73 0 //razorTransition@
+0 1 9 76 0 //razorDirectivefunctions
+0 10 1 73 0 //razorTransition{
+5 24 1 80 0 //markupTagDelimiter<
+0 1 1 81 0 //markupElementa
+0 2 5 83 0 //markupAttributeclass
+0 5 1 82 0 //markupOperator=
+0 1 1 84 0 //markupAttributeQuote"
+0 1 5 84 0 //markupAttributeQuoteThing
+0 5 1 84 0 //markupAttributeQuote"
+0 1 1 80 0 //markupTagDelimiter>
+1 28 1 73 0 //razorTransition@
+2 32 6 76 0 //razorDirective<text>
+0 6 1 73 0 //razorTransition@
+0 13 7 76 0 //markupTagDelimiter</text>
```

Comment:
Hmm, should this also be a directive?

Reply:
Correct, and it is, but I didn't update the comment I leave to remind myself what exact part and type of the text we're dealing with here, good catch.